app/components/app_patient_table_component.html.erb

```diff
@@ -18,7 +18,7 @@
         <% body.with_row do |row| %>
           <% row.with_cell do %>
             <span class="nhsuk-table-responsive__heading">Full name</span>
-            <%= patient.full_name %>
+            <%= govuk_link_to patient.full_name, patient_path(patient) %>
```

Comment:
`link_to` is configured to be the same in our app:

```suggestion
            <%= link_to patient.full_name, patient_path(patient) %>
```

Reply:
Oh that's cool, didn't realise that.